ibis/backends/flink/registry.py

```diff
@@ -0,0 +1,72 @@
+import ibis.expr.operations as ops
+from ibis.backends.base.sql.compiler import ExprTranslator
+from ibis.backends.base.sql.registry import helpers, literal_formatters
+from ibis.backends.base.sql.registry import (
+    operation_registry as base_operation_registry,
+)
+from ibis.backends.flink.utils import format_interval_literal_segments
+
+operation_registry = base_operation_registry.copy()
+
+
+def _count_star(translator: ExprTranslator, op: ops.Node) -> str:
+    return "count(*)"
```

Comment:
Since you're inheriting the base registry, `CountStar` should already be handled - I don't believe you'll need to define a handler for this op here.

Reply:
i think it compiles into something like `SELECT t0.`i`, count(1) AS `count`` instead of `count(*)`